.github/workflows/android-build-test-fabric.yml

```diff
@@ -0,0 +1,33 @@
+name: Test Android build on Fabric
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'android/**'
```

Comment:
```suggestion
      - 'android/**'
      - 'common/**'
      - 'src/fabric/**'
      - 'FabricExample/**'
      - 'package.json'
      - 'RNScreens.podspec'
```

Reply:
We would need to add `package.json` because it can affect codegen. 
What do you think about also adding `FabricExample` here?